Documentation/BuildInstructionsWindows.md

```diff
@@ -51,3 +51,22 @@ following command in an elevated PowerShell session: \
 You may have to reboot after enabling the WHPX feature.
 
 Afterwards you can start the VM with `Meta/serenity.sh run` as usual.
+
+## Note on Microsoft Store Ubuntu
+
+The Microsoft Store does not have images of Jammy Ubuntu.
+
+You'll need to add a Personal Package Archive to install the required [Prerequisites](BuildInstructionsWindows.md).
```

Comment:
This link takes you to BuildInstructionsWindows.md, where we already are. I think you meant the `Prerequisites` section of the main build instructions:
```suggestion
You'll need to add a Personal Package Archive to install the required [Prerequisites](BuildInstructions.md#prerequisites).
```